src/sentry/sentry_metrics/indexer/strings.py

```diff
@@ -182,9 +182,9 @@
     "d:spans/exclusive_time_light@millisecond": PREFIX + 406,
     "d:spans/frames_frozen@none": PREFIX + 407,
     "d:spans/frames_slow@none": PREFIX + 408,
-    "d:spans/http.response_content_length@none": PREFIX + 409,
-    "d:spans/http.decoded_response_body_length@none": PREFIX + 410,
-    "d:spans/http.response_transfer_size@none": PREFIX + 411,
+    "d:spans/http.response_content_length@byte": PREFIX + 409,
+    "d:spans/http.decoded_response_body_length@byte": PREFIX + 410,
+    "d:spans/http.response_transfer_size@byte": PREFIX + 411,
```

Comment:
The Relay counterpart has not been merged yet, so it should be OK to use the same index for the updated metric names.